app/controllers/angajati_controller.rb

```diff
@@ -0,0 +1,35 @@
+class AngajatiController < ApplicationController
+  rescue_from ActionController::ParameterMissing, with: :handle_missing_params
+
+  def index
+    @angajati = Angajat.all
+    render json: @angajati
+  end
+
+  def create
+    @angajat = Angajat.new(angajat_params)
+
+    if @angajat.save
+      render json: @angajat, status: :created
+    else
+      render json: @angajat.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @angajat = Angajat.find(params[:id])
+    @angajat.destroy
+    head :no_content
+  end
+
+  private
+
+  def angajat_params
+    params.require(:angajati).permit(:name, :position, :department)
```

Comment:
You used English for the model attributes but not for the model itself. This combination of Romanian and English can become very confusing, even for you, not to mention for someone who reads your code.